### ISSUE TITLE
Rocket.chat hang up, only reinistall helps

### ISSUE THREAD

**Issue by schebeko-cv** [RocketChat/Rocket.Chat#32000]
Hi, i have a problem and didn't know how to solve it.
I change to a chat with one of my colleges and then rocket.chat start reload the chat window.
But never ends and i can't change channel / chat anymore.
I try to clear cache or do any other way to solve it. Also after stop and restart the app, it stucked in this chat and never stops reloading.
Only way i found out is to reinstall the app and delete all user data.

Any hints or advices how we can fix this?

rocket.chat 3.9.14, Windows 11 newest updates (date 15.02.2024) 

Kind regards
Jan

**Comment by schebeko-cv** [comment]
Hier the Logs from the DevTools i found reasently.
[Rocket_chat_logs_DevTools-1710494701986.txt](https://github.com/RocketChat/Rocket.Chat/files/14613450/Rocket_chat_logs_DevTools-1710494701986.txt)

**Comment by schebeko-cv** [comment]
resently i found out the i can reset the status of the app via  Help->reset App-data. But ever i go to this chat with my college the effect is the same all others seems to work

**Comment by schebeko-cv** [comment]
The concret chat i can post the Link to it. Also when i try to open it inside the browser then its the same reacting.

**Comment by amsnek** [comment]
I think i have the same issue since Rocket.Chat Server version 6.6.0. Some Channels/Chats/Rooms will cause freeze + high cpu load  -> the Browser/Electron client needs to stopped and server removed/re-added. Browser Console gives no Hint. 
Downgrading to 6.5.4 solves the issue. I was unable to pinpoint the cause or find something specific to the Rooms/Chats in question. Initially i thought it might be related to (historic) Jitsi Calls or Polls from the poll up in there -> but it seems not.

**Comment by Gummikavalier** [comment]
If you are able to post on the channel that freezes with some client, you may be able to track the message down by doing following:

1. Type lots of new messages on the troublesome channel channel (~50) with any client that does not freeze.
2. With the client that freezed before, go to channel. (If it still immediately freezes, clean the cache first or type more messages to the channel at point 1.)
3. Slowly scroll up until the client freezes
4. The message that causes the freeze is usually the one _above_ to the last message you see in the client.

**Comment by amsnek** [comment]
Hello @Gummikavalier 
Sadly, a channel does either freeze instantly or it does not. Browser or Electron Client does make no difference. I cant even get to scrolling, the freezing (and high cpu load) happens on entering the Channel and nothing gets displayed (except the "dotting" animation). The Channels in Question arent event that large, since this is tested on a rarely used QS Instance. 
I have currently given up on tracking this, but will try again with newer updates in 6.6.x to see if anything changed. 
It also happens on entering previous direct messages with some users. 

I have rolled back and will try Updates once I can dedicated some time into this again

**Comment by tom666-debug** [comment]
Same here ... interestingly only one user (and only one channel "rocket.cat" ) is affected by this. 
Nothing suspicious found in the logs.

**Comment by Gummikavalier** [comment]
@amsnek Ok. That makes it more difficult to troubleshoot than usual.

In such situation I'd then create a migration (webhook) onto the channel, and I'd fill up the channel with the padding messages using that webhook with curl-command.

For what it is worth here is how one can try figuring out if there are messages directly from the database:
```
Increase output limit in mongo shell:
DBQuery.shellBatchSize = 100000

check the ID number of the channel:
db.rocketchat_room.find({'fname':'channelname'},{_id:1})
xxxxxxxx

Use the ID number to pring out the messages:
db.rocketchat_message.find({'rid':'xxxxxxxx'},{'ts':1,'u.name':1,'msg':1,'attachments.text':1,'file.name':1}).sort({'ts':1}) 
```

**Comment by amsnek** [comment]
@Gummikavalier Thanks! This is probably a stupid question, but I suppose this should be done after an update to 6.6.x? 🙈 
I will most likely dig into this next week, currently stuck on 6.5.x

**Comment by jpjoux** [comment]
Workaround: Downgrading to RC Server version 6.5.4 solves the issue

**Comment by rischii** [comment]
I can report that a downgrade from 6.6.6 to 6.5.4 fixed the issue for us.
We upgraded to 6.6.6 last week and gradually more and more users with broken clients.

Clients with 3.9.8 & 3.9.14 were running fine directly after the downgrade.

This should really be fixed before 6.5.4 is EOL in June!

**Comment by schebeko-cv** [comment]
We use the Cloud Version and didn't hosted it ourself. So we can't downgrade ! Please fix this on the Cloud Service: ROCKET-CHAT Support!

**Comment by reetp** [comment]
Does this still occur on 6.9.x ?

Which client or browser?

What do the server logs say?

(If yoi have a paid service eg Pro or Enterprise contact support directly)

**Comment by schebeko-cv** [comment]
With the new version 4.0 is this problem not available and is solved. Can be closed.

**Comment by schebeko-cv** [comment]
Ist Done and closed with 4.0

**Comment by amsnek** [comment]
> Ist Done and closed with 4.0
4.0? which version do you mean exactly?

**Comment by schebeko-cv** [comment]
Rocket.chat Cloud Version Hosted by rocket... 4.0.0
